samples/net/wpanusb/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _wpanusb-sample:
+
+WPANUSB sample
+##############
+
+Overview
+********
+
+This application exports ieee802154 radio over USB to be used in other
+Operation Systems like Linux. In this scenario Linux SoftMAC driver would
+be used implementing ieee802154 stack inside Linux.
+
+Requirements
+************
+
+- Linux machine host connected via USB to Zephyr WPANUSB enabled MCU
+- WPANUSB kernel driver (in the process of being open sourced)
+- IWPAN host tools at [http://wpan.cakelab.org/releases/](http://wpan.cakelab.org/releases/)
+
+Building and Running
+********************
+
+There are configuration files for various setups in the
+samples/net/wpanusb directory:
+
+- :file:`prj.conf`
+  This is the standard default config.  This can be used by itself for
+  hardware which has native 802.15.4 support.
+
+- :file:`overlay-cc1200.conf`
+  This overlay config enables support for CC1200
+
+- :file:`overlay-cc2520.conf`
+  This overlay config enables support for CC2520
+
+- :file:`overlay-mcr20a.conf`
+  This overlay config enables support for MCR20A
+
+Build the WPANUSB sample like this:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/wpanusb
+   :board: <board to use>
+   :conf: <config file to use>
+   :goals: build
+   :compact:
+
+Example building for the Nordic nRF52840 Development Kit:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/wpanusb
+   :host-os: unix
+   :board: nrf52840_pca10056
+   :goals: run
+   :compact:
+
+Example building for the Quark SE C1000 Devboard with TI CC2520 support:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/wpanusb
+   :host-os: unix
+   :board: quark_se_c1000_devboard
+   :conf: "prj.conf overlay-cc2520.conf"
+   :goals: run
+   :compact:
+
+When connected to Linux with wpanusb kernel driver, it is recognized as:
+
+.. code-block:: console
+
+   ...
+   T:  Bus=01 Lev=02 Prnt=02 Port=00 Cnt=01 Dev#=  3 Spd=12  MxCh= 0
+   D:  Ver= 1.10 Cls=ff(vend.) Sub=00 Prot=00 MxPS=64 #Cfgs=  1
+   P:  Vendor=8086 ProdID=ff03 Rev=01.00
```

Comment:
It is no longer Intel VID/PID, please change this line to `P:  Vendor=2fe3 ProdID=0101 Rev=01.00`

Reply:
Nice catch.  Fixing.